app/models/health/agency_performance.rb

```diff
@@ -18,6 +18,23 @@ def self.url
       'warehouse_reports/health/agency_performance'
     end
 
+    F2F_WINDOW = 60.days
+    COMPLETION_WINDOW = 30.days
+    RENEWAL_WINDOW = 365.days
+    WELLCARE_WINDOW = 12.months
+
+    DESCRIPTIONS = {
+      without_required_qa: 'Patients who have not completed intake and have not received a QA in the specified date range.',
+      without_required_f2f_visit: "Patients who have not received a face-to-face visit in the #{F2F_WINDOW.inspect} preceding the specified end date.",
```

Comment:
Did you mean `.inspect` or `.to_s`

Reply:
Huh, I wonder if there's a different way to get to a human readable version, inspect feels like a developer tool, but seems to do the right thing.
```
60.days.to_s
=> "5184000"
60.days.inspect
=> "60 days"
irb(main):003:0> 
```

Reply:
Yeah -- it felt to me like `inspect` wasn't supposed to go in production code -- but, I couldn't find another way to get the human readable version of a duration. There is **in_**_unit_, but it returns a float, and you need to know what unit it is in:
```
1.month.in_days
=> 30.436875
```
vs `inspect` Which figures out the parts, and which ones to display:
```
(1.month + 1.day).inspect
=> "1 month and 1 day"